src/Transform/ONNX/ConstProp.cpp

```diff
@@ -664,16 +666,16 @@ Value ConstPropGemm(PatternRewriter &rewriter, Value replacingValue,
   if (hasBias) {
     ElementsAttr bias = getConstValueElements(biasMatrixValue);
     if (beta != 1.0) {
```

Comment:
Isn't the neutral element for bias `0`? Maybe a copy/paste error from `alpha`

Reply:
I think the implementation is correct: you can skip the scalar multiplication with beta if beta==1.0

the `if (hasBias)` condition above skips adding beta*bias if bias is none - one could consider also skipping if beta==0.0, i.e., `if (hasBias && beta != 0.0)` if we ignore NaNs in the bias matrix